src/Network/Send/bRO.pm

```diff
@@ -51,7 +35,7 @@ sub new {
 	
 	while (my ($k, $v) = each %packets) { $handlers{$v->[0]} = $k}
 	$self->{packet_lut}{$_} = $handlers{$_} for keys %handlers;
-	$self->cryptKeys(46217982, 1043542121, 1807761116);
+	$self->cryptKeys(0x76181727, 0x225115EE, 0x0AB05DF3);

```

Comment:
FYI, this can be in `servers.txt` with the option `sendCryptKeys`: https://github.com/OpenKore/openkore/blob/master/tables/servers.txt#L140

Reply:
we need to update the wiki, i really forgot this option, i think the lututui too =p

Reply:
Nah, I just didn't have the time to automate it

Reply:
I wrote this to update iRO's `sendCryptKeys`, but it should work for any server:

```Perl
sub update_cryptkeys {
    my ( $file, $block, $cryptkeys ) = @_;

    return if !-f $file;

    open FP, '<', $file;
    my @lines = <FP>;
    close FP;

    # Find the first and last line of the given block.
    my $line1 = -1;
    my $line2 = -1;
    my $linec = -1;
    foreach ( 0 .. $#lines ) {
        my $line = $lines[$_];
        $line1 = $_ if $line =~ /^\[\Q$block\E\]/;
        $line2 = $_;
        $linec = $_ if $line1 > -1 && $line =~ /^\s*sendCryptKeys /;
        last if $line1 > -1 && $line1 != $_ && substr( $line, 0, 1 ) eq '[';
    }
    if ( $line1 != -1 && $linec == -1 ) {
        $linec = $line2;
        $linec-- while $lines[ $linec - 1 ] =~ /^\s*(#|$)/os;
        splice @lines, $linec, 0, '';
    }
    if ( $linec ) {
        $lines[$linec] = sprintf "sendCryptKeys 0x%s, 0x%s, 0x%s\n", uc $cryptkeys->[0], uc $cryptkeys->[1], uc $cryptkeys->[2];
    }

    if ( $linec > -1 ) {
        open FP, '>', $file;
        print FP $_ foreach @lines;
        close FP;
    }
}
```